internal/services/formatters/csharp/dotnet_cli/formatter.go

```diff
@@ -154,22 +154,31 @@ func (f *Formatter) parseFieldByIndex(index int, fieldValue string, dependency *
 func (f *Formatter) newVulnerability(dependency *dotnetDependency,
 	projectSubPath string) (*vulnerability.Vulnerability, error,
 ) {
-	code, filePath, line, err := f.getCodeFilePathAndLine(dependency, projectSubPath)
+	code, absFilePath, line, err := f.getCodeFilePathAndLine(dependency, projectSubPath)
 	if err != nil {
 		return nil, err
 	}
+
+	relFilePath, err := filepath.Rel(f.GetConfigProjectPath(), absFilePath)
+	if err != nil {
+		return nil, err
+	}
+
```

Comment:
Why this change is needed?

Reply:
Same as https://github.com/ZupIT/horusec/pull/996#discussion_r814818220